core/services/pipeline/orm_test.go

```diff
@@ -238,8 +244,11 @@ func TestORM(t *testing.T) {
 
 			test := test
 			t.Run(test.name, func(t *testing.T) {
-				orm := pipeline.NewORM(db, config)
-				jobORM := job.NewORM(db, config, orm)
+				eventBroadcaster := postgres.NewEventBroadcaster(config.DatabaseURL(), 0, 0)
+				eventBroadcaster.Start()
+				defer eventBroadcaster.Stop()
+				orm := pipeline.NewORM(db, config, eventBroadcaster)
```

Comment:
```suggestion
				orm, cleanup := cltest.NewPipelineORM(db, config)
				defer cleanup()
```
I see these setup lines in a bunch of tests and I'm wondering if it'd be worth making a factory to cutdown on setup boilerplate?

Reply:
Agree with @se3000 about factory function